exam_app/views.py

```diff
@@ -79,6 +94,9 @@ def addQuestion(request, exam_id):
 
 
 def viewAllExamsInstructors(request):
+    if request.method == 'POST':
+        delete_exam = request.POST['delete_exam']
+        MakeExam.objects.get(id = delete_exam).delete()
```

Comment:
Here you are only deleting the exam objects and the questions related to the exam are still in the database. (In this case, you need to delete the related questions 1st)

Use this code before line 99
MakeQuestion.objects.filter(exam_model__id=delete_exam).delete()